### PR TITLE
MAGN-8388 Toggling the Dynamo Background Preview is not sticky "inter-session"

### PR DESCRIPTION
**Pull request by ikeough**
### Purpose

Fix [MAGN-8388](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8388) by doing the following:
- Remove an extraneous property `Watch3DViewModelStartupParams.IsActiveAtStart` which was always being set to true, and was unneccessary because we already get a reference to an `IPreferences` object which can be used to get the `IsBackgroundPreviewActive` property.
- Set `IPreferences.IsBackgroundPreviewActive` when `Watch3DViewModelBase.Active` is set.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
  - No new strings added.
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
  - No new strings added.
### Reviewers

@mjkkirschner

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Watch3D/Watch3DViewModelBase.cs [hunk 1]
```diff
@@ -25,7 +25,6 @@ public class Watch3DViewModelStartupParams
         public IRenderPackageFactory RenderPackageFactory { get; set; }
         public IDynamoViewModel ViewModel { get; set; }
         public INotifyPropertyChanged RenderPackageFactoryViewModel { get;set; }
-        public bool IsActiveAtStart { get; set; }
```
Comment (ikeough) [inline src/DynamoCoreWpf/ViewModels/Watch3D/Watch3DViewModelBase.cs]: This property is not required as we can already get at the pref. setting through the `IPreferences` object.

**Comment by ikeough** [comment]
@mjkkirschner ping!

**Comment by mjkkirschner** [comment]
@ikeough besides the merge conflict, the solution looked good.

**Comment by mjkkirschner** [comment]
@ikeough please fix the merge conflict, then will review

**Comment by ikeough** [comment]
@mjkkirschner If this needs another Review, let me know.

**Comment by mjkkirschner** [comment]
@ikeough looks good to merge, tested it out and works.

**Pull request merged**